index.html

```diff
@@ -13843,6 +13843,7 @@ <h3>Presentational Roles Conflict Resolution</h3>
 <span class="comment">&lt;!-- 2. [role="presentation"] negates both the implicit 'heading' and the non-global aria-level. --&gt;</span>
 &lt;h1 role="presentation" aria-level="2"&gt; Sample Content &lt;/h1&gt;
 		</pre>
+		<p>Authors MUST NOT use <rref>presentation</rref> and <rref>none</rref> on elements where users agents will ignore that role because it conflicts with one of the above items.</p>
```

Comment:
```suggestion
		<p>Authors MUST NOT use <rref>presentation</rref> and <rref>none</rref> on elements where user agents will ignore that role because it conflicts with one of the above items.</p>
```